addons/editor-unshare-button/userscript.js

```diff
@@ -9,6 +9,8 @@ export default async function ({ addon, msg, global, console }) {
     button.classList.add("sa-unshare-button");
     button.querySelector("span").innerText = msg("unshare-button");
     button.addEventListener("click", async (e) => {
+      // Prevent the listener if unshared
+      if (!e.target.closest("sa-unshare-button")) return;
```

Comment:
The condition is always evaluating to `false`. Did you forget a dot before `sa-` to refer to the class name?
And if you fix that, won't the condition be met every time?
I don't understand why this line was added.

Reply:
@Norbiros Could you explain?

Reply:
There was bug with 2 confirmations in unshare button in editor (one from editor-unshare and from confirm-actions).

> Did you forget a dot before sa- to refer to the class name?

Yes! I forgot about it...

> And if you fix that, won't the condition be met every time?

Nope, when you are using it without confirmations it always returns true. But look at this example:
User clicks unshare button in editor >> it shows 2 popups >> when you click first one it unshares project (and changes button from sa-unshare, to normal share button) >> but when you click second one it executes .click on the same button as first one, so it shares project
So I added that IF to check if button didn't change.

Reply:
@Norbiros Wow, that's not what I expected would be the reason for adding that one line, and I think it would be better to fix that issue in a different way. Or the code comment above that line needs to explain this in detail.

Reply:
It would be easier to understand if a similar condition was added to the `confirmations` code instead.

Reply:
Confirm actions always runs first (to fix other bugs), so we can't fix that (using js). Maybe we can look for classes, and add `:not`?

Reply:
editor-unshare-button is buggy even if confirmations is disabled.

Reply:
I'll be fixing this in a separate PR, since the bug isn't really specific to confirm-actions.